_includes/header.html

```diff
@@ -1,46 +1,93 @@
-<header class="site-header">
-	<div class="branding">
-		{% if site.theme_settings.gravatar %}
-		<a href="{{ site.baseurl }}/">
-			<img class="avatar" src="https://secure.gravatar.com/avatar/{{ site.theme_settings.gravatar }}?s=100" alt="Bits & Bytes Home"/>
-		</a>
-		{% elsif site.theme_settings.avatar %}
-		<a href="{{ site.baseurl }}/">
-			<img class="avatar" src="{{ site.baseurl }}/{{ site.theme_settings.avatar }}" alt="Bits & Bytes Home"/>
-		</a>
-		{% endif %}
-		<a class="site-title" href="{{ site.baseurl }}/">{{ site.theme_settings.title }}
-			{% if site.theme_settings.subtitle %}
-				<br /><span class="site-subtitle">{{ site.theme_settings.subtitle }}</span>
-			{% endif %}
-		</a>
-	</div>
-	<nav class="site-nav">
-		<ul>
-			{% for page in site.pages %}
-			{% if page.title and page.hide != true %}
-			<li>
-				<a class="page-link" href="{{ page.url | relative_url }}">
-					{{ page.title }}
+<header class="header header--blue">
+	<div class="wrapper">
+	  <div class="container flex">
+		<div class="header__brand header__brand--text">
+		  <a href="/" class="header__brand-title">Bits & Bytes</a>
```

Comment:
I'm seeing this misaligned on mobile (see screenshot) - is anyone else seeing this too?
<img width="499" alt="Screen Shot 2024-07-22 at 10 45 22 AM" src="https://github.com/user-attachments/assets/7c837fbe-2ae3-4af8-960b-801f2b5f2f61">

Reply:
Hey, I'm seeing this as well, but I'm not sure what's causing this.

Do you see anything @HaSistrunk